src/appleseed/foundation/math/beziercurve.h

```diff
@@ -72,19 +72,26 @@ class BezierCurveBase
     typedef Vector<T, 3> VectorType;
     typedef AABB<T, 3> AABBType;
     typedef Matrix<T, 4, 4> MatrixType;
+    typedef Color<T, 3> ColorType;
 
     // Degree of the Bezier curve.
     static const size_t Degree = N;
 
     // Constructors.
     BezierCurveBase();
-    BezierCurveBase(const VectorType ctrl_pts[N + 1], const ValueType width);
-    BezierCurveBase(const VectorType ctrl_pts[N + 1], const ValueType width[N + 1]);
-    BezierCurveBase(const BezierCurveBase& curve, const MatrixType& xfm);
+    BezierCurveBase(const VectorType ctrl_pts[N + 1], const ValueType width,
```

Comment:
Check how we indent multi-line methods/constructors and indent similarly (fix everywhere).